shared/src/business/useCases/docketEntry/addPaperFilingInteractor.test.ts

```diff
@@ -41,7 +41,7 @@ describe('addPaperFilingInteractor', () => {
 
     applicationContext
       .getUseCases()
-      .getFeatureFlagValueInteractor.mockReturnValue(Promise.resolve(true));
+      .getAllFeatureFlagsInteractor.mockReturnValue(Promise.resolve(true));
```

Comment:
🤔 hm, wouldn't these change to an `object` instead of a `boolean`? Maybe I'm missing something.

Reply:
Good catch, you're totally right! Updating all the tests that do this